test/src/unit/repo/core/model/bson/ut_repo_node_metadata.cpp

```diff
@@ -30,13 +30,12 @@ using namespace repo::core::model;
 static MetadataNode makeRandomMetaNode()
 {
 	uint32_t nItems = rand() % 10 + 1;
-	std::vector<std::string> keys, values;
+	std::unordered_map<std::string,repo::lib::RepoVariant> metaDataUnMap;
```

Comment:
same here, now we support a variant, the data should not be just strings.

Reply:
Now we have changed this to integer and are testing this.